app/experimenter/targeting/constants.py

```diff
@@ -802,7 +802,12 @@ def __post_init__(self):
     name="Existing Windows 7+ user",
```

Comment:
Let's update the name and description to make it clear this doesn't include users signed in to FXA sync